roles/tas_single_node/tasks/main.yml

```diff
@@ -1,4 +1,7 @@
 ---
+- name: Check OS System Support
```

Comment:
Nit: sorry I haven't noticed this before, but let's just use "OS" or "operating system". I will approve the PR, but please fix this before merging. Thanks!

Reply:
Yes, good point - I will fix that :)